### PR TITLE
Asking for help lesson: Update verbiage

### LINKED ISSUE #23928: Style Guide - Foundations/Introduction: Learning Outcomes and Knowledge Checks
# Foundations/Introduction Section

This issue is part of the Style Guide Community Project, a way of offering members of The Odin Project community a chance to contribute to our curriculum. Users who wish to take part and contribute to any Style Guide issues should have thoroughly read the [layout style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md) and [The Odin Project contributing guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md).
  
## Description of Issue

This issue ensures the Learning Outcomes and Knowledge Check sections have the correct verbiage and formatting. If one of these sections does not exist in a lesson, it should not be added.

Each applicable lesson for this issue should meet the following requirements:

- The Learning Outcomes section is renamed to "Lesson Overview".
- The Lesson Overview section follows the below format:
	```
	### Lesson Overview

	This section contains a general overview of topics that you will learn in this lesson.

	*   LESSON OVERVIEW ITEMS...
	```
- The Knowledge Check section follows the below format:
	```
	### Knowledge Check

	This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.

	*   KNOWLEDGE CHECKS...
	```
- Each Knowledge Check item follows the below format:
	```
	*   <a class="knowledge-check-link" href="A KNOWLEDGE CHECK URL">A KNOWLEDGE CHECK QUESTION.</a>
	```
- Each Knowledge Check item redirects to a resource mentioned in the lesson, or to a section within the lesson, where the answer can be found. 
	- If it doesn't, note this in your PR, but don't remove the item.
  
## Acceptance Criteria

- [ ] The following lessons are updated according to the issue description above:
	- [ ] [Asking For Help](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/asking-for-help)
	- [ ] [How Does the Web Work?](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/how-does-the-web-work)

### PR DESCRIPTION
**Pull request by nguyens20**
Updated the verbiage and formatting for the Asking for help lesson in the Foundations course.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #23928 

**2. This PR:**
Changed Learning Outcomes to Lesson Overview
Update the format for some of the Knowledge Check items.

**3. Additional Information:**

### CODE REVIEW

File: foundations/introduction/asking_for_help.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Along your programming journey, you will inevitably need to ask questions in an online coding community like ours, or in conversations with your colleagues. While asking questions may seem rather simple, in a programming environment it is essential to ask well formed questions with as much context as needed so you can get the most out of asking for help. In essence, “help others help you!” This lesson will provide you with the tools to effectively ask questions in communities such as our Discord chat, Stack Overflow, and in the workplace!
 
-### Learning Outcomes
+### Lesson Overview
 By the end of this lesson, you should be able to do the following:
```
Comment (thatblindgeye) [inline foundations/introduction/asking_for_help.md:6]: The line below the heading should also be updated.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: foundations/introduction/asking_for_help.md (hunk 1 above)
Comment (nguyens20) [reply foundations/introduction/asking_for_help.md:6]: Gotcha !

Comment (nguyens20) [reply foundations/introduction/asking_for_help.md:6]: I made a new PR #23968 to update the line below the heading as well. Its my first time contributing so if I'm doing anything wrong please let me know !

**Review by thatblindgeye — APPROVED** [review]
Ah disregard my original comment. Just saw your comments on the issue itself. Will reclose this and go to the most recent PRs.

**Pull request closed without merging**